lockbox-ios/View/ItemDetailView.swift

```diff
@@ -16,19 +16,25 @@ struct ItemDetailCellConfiguration {
     let password: Bool
     let valueFontColor: UIColor
     let accessibilityId: String
+    let copyButton: Bool
+    let openButton: Bool
 
     init(title: String,
          value: String,
          accessibilityLabel: String,
          password: Bool,
          valueFontColor: UIColor = UIColor.black,
-         accessibilityId: String) {
+         accessibilityId: String,
+         copyButton: Bool = false,
+         openButton: Bool = false) {
```

Comment:
Nit: I appreciate you already copying the given pattern. I would prefer to see a `show` or `has` prefix for these `Boo`s.